lms/templates/dashboard/_dashboard_xseries_info.html

```diff
@@ -0,0 +1,36 @@
+<%page args="course_program_info, program_marketing_url, enrollment_mode" />
+<%!
+    from urlparse import urljoin
+    from django.utils.translation import ugettext as _
+%>
+<%namespace name='static' file='../static_content.html'/>
+<% program_marketing_url = urljoin(program_marketing_url, course_program_info['category'] +'/{}').format(course_program_info['marketing_slug'])  %>
```

Comment:
can we construct this link inside the `_get_xseries_programs` function instead?  that would be easier to unit test, and would simplify the template logic.